py-polars/polars/series/list.py

```diff
@@ -221,6 +286,25 @@ def unique(self, *, maintain_order: bool = False) -> Series:
         maintain_order
             Maintain order of data. This requires more work.
 
+        Examples
+        --------
+        >>> s = pl.Series([[2, 1, 2], [5, 4, 5]])
+        >>> s.list.unique()
```

Comment:
This one should be skipped as it may fail (the result is not guaranteed to be in this order).
```suggestion
        >>> s.list.unique()  # doctest: +SKIP
```